python/semantic_kernel/connectors/ai/google/google_ai/services/google_ai_text_completion.py

```diff
@@ -0,0 +1,221 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import sys
+from collections.abc import AsyncGenerator
+from typing import TYPE_CHECKING, Any
+
+import google.generativeai as genai
+from google.generativeai import GenerativeModel
+from google.generativeai.protos import Candidate
+from google.generativeai.types import AsyncGenerateContentResponse, GenerateContentResponse, GenerationConfig
+from pydantic import ValidationError
+
+from semantic_kernel.connectors.ai.google.google_ai.google_ai_prompt_execution_settings import (
+    GoogleAITextPromptExecutionSettings,
+)
+from semantic_kernel.connectors.ai.google.google_ai.services.google_ai_base import GoogleAIBase
+from semantic_kernel.connectors.ai.text_completion_client_base import TextCompletionClientBase
+
+if sys.version_info >= (3, 12):
+    from typing import override  # pragma: no cover
+else:
+    from typing_extensions import override  # pragma: no cover
+
+from semantic_kernel.connectors.ai.google.google_ai.google_ai_settings import GoogleAISettings
+from semantic_kernel.contents import TextContent
+from semantic_kernel.contents.streaming_text_content import StreamingTextContent
+from semantic_kernel.exceptions.service_exceptions import ServiceInitializationError
+
+if TYPE_CHECKING:
+    from semantic_kernel.connectors.ai.prompt_execution_settings import PromptExecutionSettings
+
+
+class GoogleAITextCompletion(GoogleAIBase, TextCompletionClientBase):
+    """Google AI Text Completion Client."""
+
+    def __init__(
+        self,
+        ai_model_id: str | None = None,
+        api_key: str | None = None,
+        service_id: str | None = None,
+        env_file_path: str | None = None,
+        env_file_encoding: str | None = None,
+    ) -> None:
+        """Initialize the Google AI Text Completion Client.
+
+        If no arguments are provided, the service will attempt to load the settings from the environment.
+        The following environment variables are used:
+        - GOOGLE_AI_AI_MODEL_ID
+        - GOOGLE_AI_API_KEY
+
+        Args:
+            ai_model_id (str | None): The AI model ID. (Optional)
+            api_key (str | None): The API key. (Optional)
+            service_id (str | None): The service ID. (Optional)
+            env_file_path (str | None): The path to the .env file. (Optional)
+            env_file_encoding (str | None): The encoding of the .env file. (Optional)
+
+        Raises:
+            ServiceInitializationError: If an error occurs during initialization.
+        """
+        try:
+            google_ai_settings = GoogleAISettings.create(
+                ai_model_id=ai_model_id,
```

Comment:
Similar question: in other connectors this attribute is `text_model_id`. Should we align?